packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1542,12 +1566,25 @@ export default abstract class SqlIntegration
       displayNameColumn,
       includesPagesTable,
       includesScreensTable,
+      groupByColumns,
+      eventHasUniqueTable,
     } = this.getSchemaFormatConfig(schemaFormat);
 
     const currentDateTime = new Date();
-    const SevenDaysAgo = new Date(
+    const sevenDaysAgo = new Date(
       currentDateTime.valueOf() - 7 * 60 * 60 * 24 * 1000
     );
+    const startYear = sevenDaysAgo.getFullYear();
```

Comment:
This is very verbose - open to suggestions

Reply:
We have the `date-fns` package available on the back-end. It should help accomplishing this easier

- https://date-fns.org/v2.30.0/docs/sub 
- https://date-fns.org/v2.30.0/docs/format

Reply:
@bttf Great suggestion - thank you! I've pushed up changes utilizing `date-fns` - way less verbose and way better readability. I also added a `dateFormat` property to the `SchemaFormatConfig` for added flexibility down the road.